src/main/java/com/minecraftabnormals/abnormals_core/common/blocks/wood/StrippedWoodPostBlock.java

```diff
@@ -0,0 +1,100 @@
+package com.minecraftabnormals.abnormals_core.common.blocks.wood;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.ChainBlock;
+import net.minecraft.block.IWaterLoggable;
+import net.minecraft.block.LanternBlock;
+import net.minecraft.fluid.FluidState;
+import net.minecraft.fluid.Fluids;
+import net.minecraft.item.BlockItemUseContext;
+import net.minecraft.state.BooleanProperty;
+import net.minecraft.state.EnumProperty;
+import net.minecraft.state.StateContainer;
+import net.minecraft.state.properties.BlockStateProperties;
+import net.minecraft.util.Direction;
+import net.minecraft.util.Direction.Axis;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.shapes.ISelectionContext;
+import net.minecraft.util.math.shapes.VoxelShape;
+import net.minecraft.world.IBlockReader;
+import net.minecraft.world.World;
+
+public class StrippedWoodPostBlock extends Block implements IWaterLoggable {
```

Comment:
Wouldn't it make more sense to have the non-stripped version be the default post class?

Reply:
It might from a theoretical standpoint, but the non-stripped version has an extra method override that made it cleaner for the default post to be the stripped version. I can make the non-stripped version the default post class, but I'd likely have to copy-paste methods over instead of subclassing.

Reply:
I don't even see a need for two classes. You can achieve the same thing using one class by making `block` in `WoodPostBlock` nullable.